packages/@sanity/base/src/components/DevServerStatus.js

```diff
@@ -0,0 +1,90 @@
+import React, {PureComponent} from 'react'
+import polyfilledEventSource from '@sanity/eventsource'
+import Snackbar from 'part:@sanity/components/snackbar/default'
+
+const isWindowEventSource = Boolean(typeof window !== 'undefined' && window.EventSource)
+const EventSource = isWindowEventSource
+  ? window.EventSource // Native browser EventSource
+  : polyfilledEventSource // Node.js, IE, Edge etc
+
+const STATE_CLOSED = 0
+const STATE_OPEN = 1
```

Comment:
`0` actually means it's connecting (EventSource will automatically try to reconnect when disconnected), so the constant should reflect this. `2` means it's closed, which can happen in some cases (browser gives up reconnecting after closing laptop, or similar).

Reply:
OK! So snackbar should be displayed when it's reconnecting and closed.

Reply:
Correct